pkg/kubelet/kubelet_pods.go

```diff
@@ -675,6 +676,38 @@ func (kl *Kubelet) podIsTerminated(pod *v1.Pod) bool {
 	return false
 }
 
+func (kl *Kubelet) safeToDeletePod(pod *v1.Pod) bool {
```

Comment:
Yes, we should also do that.  Is: "kl.containerManager.NewPodContainerManager().Exists(pod)" the correct statement?